### DISCUSSION TITLE
[Q] I am not able to read these cells as merged. Can you please help me.

### DISCUSSION THREAD

**Discussion by gaurav-cointab** [exceljs/exceljs#2401] [Q&A]
Hi Team,

I am not able to read these merged cell as "merged", they are being read as "null". 

For the merged cells I am getting
    isMerged = false 
    type = 0 -- ValueType.Null instead of ValueType.Merge
    effectiveType = 0 -- ValueType.Null instead of ValueType.Merge
    value = null
    master = the cell itself and not the merged master cell
    
Can you please help me understand what I am doing wrong?

[Merged Cell Test.xlsx](https://github.com/exceljs/exceljs/files/5539949/Merged.Cell.Test.xlsx)

Forgot to mention that I am reading the excel in stream mode.

**Comment by PAllisonVSO** [comment]
I have run into this same issue when streaming a file in ExcelJS
